app/_hub/kong-inc/prometheus/index.md

```diff
@@ -185,14 +182,18 @@ allow access to the `/metrics` endpoint to Prometheus:
    Prometheus servers use to access the metric data. Access through any
    firewalls would also need to be configured.
 
-2. You can proxy the Admin API through Kong itself then use plugins to limit
+1. If the [Status API](https://docs.konghq.com/latest/configuration/#status_listen)
+   is enabled, then its `/metrics` endpoint can be used.
+   This is the preferred method for nodes configured as a Data Plane.
+
+1. You can proxy the Admin API through Kong itself then use plugins to limit
    access. For example, you can create a route `/metrics` endpoint and have
    Prometheus access this endpoint to slurp in the metrics while preventing
    others from accessing it. The specifics of how this is configured will depend
    on your specific setup. For details, see [Securing the Admin
    API](https://docs.konghq.com/latest/secure-admin-api/#kong-api-loopback).
```

Comment:
This was a workaround that we had documented in absence of `status_listen`. This should be removed now.